recipes/blast/build.sh

```diff
@@ -13,7 +13,7 @@ elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
     echo "Platform: Linux"
 
     cd $SRC_DIR/c++/
-    ./configure --prefix=$PREFIX
+    ./configure --prefix=$PREFIX --with-hard-runpath
```

Comment:
Would be nice to add a comment here as well.

Reply:
Will do.